vault/audit_broker_test.go

```diff
@@ -0,0 +1,136 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package vault
+
+import (
+	"context"
+	"crypto/sha256"
+	"testing"
+
+	"github.com/hashicorp/eventlogger"
+	"github.com/hashicorp/vault/audit"
+	"github.com/hashicorp/vault/builtin/audit/syslog"
+	"github.com/hashicorp/vault/helper/testhelpers/corehelpers"
+	"github.com/hashicorp/vault/internal/observability/event"
+	"github.com/hashicorp/vault/sdk/helper/salt"
+	"github.com/hashicorp/vault/sdk/logical"
+	"github.com/stretchr/testify/require"
+)
+
+// testBackend will create an audit.Backend (which expects to use the eventlogger).
+func testBackend(t *testing.T, path string, config map[string]string) audit.Backend {
```

Comment:
`testBackend` is a very generic name. Can this be renamed to something specific to audit?

Reply:
I agree, especially in a broad package like github.com/hashicorp/vault/vault